src/creatures/creatures_definitions.hpp

```diff
@@ -10,6 +10,8 @@
 #ifndef SRC_CREATURES_CREATURES_DEFINITIONS_HPP_
 #define SRC_CREATURES_CREATURES_DEFINITIONS_HPP_
 
+#include <pch.hpp>
+
```

Comment:
This is not necessary, it's already in the precompiled header, it will cause circular include problem.

```suggestion
```